### ISSUE TITLE
Duda sobre cajas.csv

### ISSUE THREAD

**Issue by gabo1243** [IIC2413/Syllabus-2023-1#112]
Hola, estaba revisando el archivo cajas.csv y no entiendo porque se repiten los ids. ¿Se ocupan las mismas cajas todo el tiempo?
Agradecería que pudieran explicar esto.

**Comment by NicolasOlmosQuiroga** [comment]
Exacto, las cajas que llevan los productos en cada compra en si son las mismas, pero al relacionarlas con los atributos de las otras entidades, se puede saber exactamente en que compra se usan que caja (o cajas) y que producto esta en esa caja (o cajas)

Saludos.
